bin/build-ane

```diff
@@ -15,7 +15,7 @@ mkdir -p $TEMP_PATH/default
 echo "****** Compiling AS3 library *******"
 
 "$ACOMPC" \
-    -swf-version $AIR_VERSION \
+    -swf-version $SWF_VERSION \
```

Comment:
I confused the AIR SDK version with the SWF version. I use this PR as an opportunity to fix this.